README.md

```diff
@@ -47,6 +47,11 @@ Install all dev dependencies (consider using Python virtual environments):
 pip install -r dev-requirements.txt
 ```
 
+Generate source files
+```bash
+make generate
+```
+
```

Comment:
Should this change have been made to `GET_STARTED.md` as well?  (Are you still sure you want to have this information in two places??  🙃)

Reply:
Yeah I am updating that in a followup PR, as I said in one of the previous comments, this is gonna be replaced by `pip install <package>` as soon as I can pusblish it on pypi. Therefore this is just a temporary duplication.